cime/config/e3sm/config_files.xml

```diff
@@ -199,6 +199,7 @@
       <value component="allactive">$CIMEROOT/config/$MODEL/testmods_dirs</value>
       <value component="drv"      >$CIMEROOT/src/drivers/$COMP_INTERFACE/cime_config/testdefs/testmods_dirs</value>
       <value component="cam"      >$SRCROOT/components/cam/cime_config/testdefs/testmods_dirs</value>
+      <value component="scream"   >$SRCROOT/components/scream/cime_config/testdefs/testmods_dirs</value>
```

Comment:
why do we need this? We're not using testmods, are we? I hate testmods, which I see as a remnant from before the dawn of version control.

Reply:
nope, I will remove.  See below.